exporters/stats/stackdriver/src/main/java/io/opencensus/exporter/stats/stackdriver/StackdriverExporterWorker.java

```diff
@@ -181,13 +181,16 @@ void export() {
                 .build();
         metricServiceClient.createTimeSeries(request);
         span.addAnnotation("Finish exporting TimeSeries.");
+      } catch (com.google.api.gax.rpc.UnavailableException e) {
+        if (e.getLocalizedMessage()
+            .startsWith(
+                "io.grpc.StatusRuntimeException: UNAVAILABLE: HTTP/2 error code: NO_ERROR")) {
+          continue; // Silently skip NO_ERROR Goaway
```

Comment:
Should the exporter retry in this case?

Reply:
I'm hesitant on doing retry in this case, since I think NO_ERROR Goaway indicates server should have received and processed the request with no error. Retrying might generate duplicated stats and increase the burden of server.